cmd/user-cluster-controller-manager/main.go

```diff
@@ -146,9 +148,9 @@ func main() {
 	if len(runOp.caBundleFile) == 0 {
 		log.Fatal("-ca-bundle must be set")
 	}
-	if runOp.userClusterLogging {
+	if runOp.userClusterLogging || runOp.userClusterMonitoring {
 		if runOp.mlaGatewayPort == 0 {
-			log.Fatal("-mla-gateway-port must be set when enabling user cluster logging")
+			log.Fatal("-mla-gateway-port must be set when enabling user cluster logging or monitoring")
```

Comment:
why not set some sensible default to -mla-gateway-port? This way we could avoid rage-exits.

Reply:
This `-mla-gateway-port` will be a `NodePort` assigned to the `mla-gateway` service in the Seed, and I don't know how to default it :)

Reply:
Similarly, we will do something similarly as we did for `openvpnServerPort`: https://github.com/kubermatic/kubermatic/blob/master/cmd/user-cluster-controller-manager/main.go#L103, for example, openvpnServerPort is fetched from the NodePort of `openVPN server` service:
https://github.com/kubermatic/kubermatic/blob/master/pkg/resources/data.go#L442-L455
, and we will have a `mla-gateway` service in seed, and pass NodePort from there.